src/gui/widgets/FxLine.cpp

```diff
@@ -208,6 +208,7 @@ void FxLine::contextMenuEvent( QContextMenuEvent * )
 	contextMenu->addHelpAction();
 	contextMenu->exec( QCursor::pos() );
 	delete contextMenu;
+	delete this;
```

Comment:
This always causes deletion of the FxLine, even when you don't select any option in the context menu. You can even remove master with this. You should remove this line.